### ISSUE TITLE
Fix html5_formality problems problem_838 through problem_852

### ISSUE THREAD

**Issue by perspegrity5** [perspegrity5/UMassONRMathTutorsContent#36]
LS:  "The problem, hint, and answer will all play one after the other without the student clicking the Hint button. (These worked before the system was changed.) When I tested one of the 4mality conversions with David's new test button, there was 1) echoing in the sound and 2) the hint button played the answer hint along with the other hints.  We may need to reinsert the sound in these problems and update the template....you will see the 15 Formality conversions.  They were done with the old template and aren’t behaving very well in the new system. Can you reinsert the sound files and do any other template updates needed?"

When done, you can put them into the CHECKED folder inside the 0 HTML5 folder (because they don’t need to be checked again for Edge errors).

TM: changed the problems from ready to disabled √

**Comment by perspegrity5** [comment]
these were set to  disabled

**Comment by lynn99** [comment]
Yes they should stay as disabled until Kevin finishes updating their templates.  I’ll let you know when I’ve put the repaired versions in your upload folder.

—
Lynn Stephens
Post Doctoral Researcher
School of Computer Science  and 
College of Education
University of Massachusetts—Amherst
(413) 230-4634

**Comment by lynn99** [comment]
These have been repaired but are not priority for uploading at this time.  
If any of these are needed for Affect, I can move them to Tom's upload folder.

**Comment by perspegrity5** [comment]
Lynn, I don't see any reason not to move things to my upload folder that are ready to be tested and have the answer and numHints provided to me. (but you may know of some reasons)
I am waiting to upload until there are a few in my folder.  

On Nov 5, 2014, at 11:38 AM, lynn99 notifications@github.com wrote:

> These have been repaired but are not priority for uploading at this time.
> 
> If any of these are needed for Affect, I can move them to Tom's upload folder.
> 
> —
> Reply to this email directly or view it on GitHub.

===---===---===---===---===---===---===---===---===---===---===---===---===---===---===---===---===---===---===---===
Tom Murray, EdD

Senior Research Fellow in Cognitive Tools, Educational Technology & Online Communities 

University of Massachusetts Amherst

Home: 36 Fruit Street, Northampton MA 01060; phone: is413-then-695-and-2800 

http://www.tommurray.us (also tommurray.us@gmail.com)

**Comment by lynn99** [comment]
Yeah, it’s just that I still have to check them and they aren’t priority.

—
Lynn Stephens
Post Doctoral Researcher
School of Computer Science  and 
College of Education
University of Massachusetts—Amherst
(413) 230-4634
